french/savoir_africain/savoir_africain.md

```diff
@@ -0,0 +1,2 @@
+Après avoir eu cette pensée, tout à coup un génie(africain biensûr ;)) apparait et vous propose ce dilemne hyper contraignant.
+"Veux tu vraiment [savoir ce que ferait la science ancienne africaine](science_ancienne/historique.md) ou un [plat de poulet](poulet/kfc.md) te suffirait? "
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.